config/config.default.php

```diff
@@ -88,6 +88,10 @@
     # Your privacy@ contact address
     'privacy_email' => env('PRIVACY_EMAIL', null),
 
+    # Enable checkbox to consent for storage of nick, email address and worked hours (and T-shirt size)
```

Comment:
I would prefer a shorter text here as its currently a copy of the whole translation

Reply:
do you have suggestions

Reply:
```suggestion
    // Show opt in to long term saving of personal data on user profile and registration pages
```